gravitee-apim-console-webui/src/management/api/endpoints/groups/edit/api-proxy-group-edit.component.ts

```diff
@@ -108,7 +108,10 @@ export class ApiProxyGroupEditComponent implements OnInit, OnDestroy {
           groupIndex !== -1 ? api.proxy.groups.splice(groupIndex, 1, updatedGroup) : api.proxy.groups.push(updatedGroup);
           return this.apiService.update(api.id, api);
         }),
-        tap(() => this.snackBarService.success('Configuration successfully saved!')),
+        tap(() => {
+          this.snackBarService.success('Configuration successfully saved!');
+          this.isReadOnly = true;
```

Comment:
I think that you should update the formInitialValues to make it disappear. It's not really logical to become read only when you have created something imho